riff-raff.yaml

```diff
@@ -0,0 +1,19 @@
+stacks: [flexible]
+regions: [eu-west-1]
+deployments:
+  viewer:
+    type: autoscaling
+    parameters:
+      bucket: composer-dist
+    dependencies:
+      - viewer-ami-update
+  viewer-ami-update:
+      type: ami-cloudformation-parameter
+      parameters:
+        amiTags:
+          Recipe: editorial-tools-basic
+          AmigoStage: PROD
```

Comment:
That's one of the tag added to an AMI by Amigo. I would work without it but it is an additional way to make sure riff-raff discovers the right AMI